cwms-data-api/src/test/resources/cwms/cda/data/dto/timeseriesprofile/timeseriesprofileinstance.json

```diff
@@ -0,0 +1,41 @@
+{
+  "time-series-profile":
+  {
+    "location-id": "DET",
+    "key-parameter": "Depth",
+    "parameter-list": [ "Temp-Water", "Pres", "%-DO",  "Conc-DO", "Cond", "pH", "TurbN",  "Depth","Volt-Battery"],
+    "description": "Demonstration Profile for Couger Lake",
+    "office-id": "SWT",
+    "ref-ts-id": "DET.Elev-Forebay.Inst.0.0.Mixed-Rev"
+  },
+
+  "value-list" :  [
+    {
+      "time-zone": "US/Pacific",
+      "times" : ["09/09/2019,13:16:01", "09/09/2019,13:17:20"],
```

Comment:
CDA *always* does either UNIX Epoch UTC or Full ISO 8601 Date-Time-TZ, but we still prefer to render everything as UTC.

Reply:
we can use the full iso and get rid of the time-zone tag

Reply:
removed the timezone, added timezone to the ISO8601 times

Reply:
Generally your DTO will use (preferable) Instants or (second choice) ZonedDateTime and then jackson will just do the right with regard to format.   In the text-timeseries work we found that current version of jackson accepts epoch millis or ISO8601 when deserializing Instant which is user friendly.   Serializing Instant into epoch millis isn't very sexy but its short, efficient and rarely confused.